### ISSUE TITLE
add thermos fixes by gambiarra

### ISSUE THREAD

**Issue by Pa4ok** [Funwayguy/BetterQuesting#697]
how about add gambiarra fixes for betterquesting & thermos 1.7.10
[link](https://github.com/juanmuscaria/Gambiarra-BetterQuesting)

**Comment by Funwayguy** [comment]
Not my job to fix other peoples mods let alone an unsupported hackjob of a server.

Bukkit, Cauldron, Thermos, Feather, Sponge, or whatever derivative of those will never be supported. If a server owner chooses to use them then that's on them to deal with the consequences.

**Comment by Pa4ok** [comment]
but you don’t even have to make a fix, just transfer, it will remove a lot of questions in the future
i don't think so many admins use pure forge

**Comment by Funwayguy** [comment]
Just because a bunch of admins continue to duct tape Bukkit plugins to Forge despite repeated warnings doesn't mean anyone should be expected to support it and the multitude of problems that occurs because of it. I know many mod devs who similarly don't support these setups. Accepting this 'patch' would cause more questions and problems because this wouldn't be the the first or last issue that BQ has been incorrectly blamed for yet expected to fix. Am I supposed to provide patches for every little thing these server hacks break? What's stopping server admins from having server side only Forge mods made that achieve the same thing?

TLDR: No. Never.
